io/zenoh-transport/tests/unicast_transport.rs

```diff
@@ -375,7 +404,14 @@ async fn open_transport(
     // Open transport -> This should be accepted
     for e in client_endpoints.iter() {
         println!("Opening transport with {}", e);
-        let _ = ztimeout!(client_manager.open_transport(e.clone())).unwrap();
+        //let _ = ztimeout!(client_manager.open_transport(e.clone())).unwrap();
+        let e = client_manager.open_transport(e.clone()).await;
+        match e {
```

Comment:
In tests is fine to `unwrap()`. We early catch the problem and we do not continue in case of failure.
In this particular case, if the `open_transport` fails, all the following checks will fail as well. Better be explicit with an `unwrap()` which also return the line number where the error occured.

Reply:
That was just the debug code slipped here, sorry